packages/core/src/resolver.js

```diff
@@ -19,15 +18,18 @@ export default class Resolver {
 
     // ESM Loader
     this.esm = esm(module)
+
+    // module resolver
+    this.moduleResolver = moduleResolver || require.resolve
```

Comment:
I think we may not need `moduleResolver` for consistency if `require.resolve` works fine 😄

Reply:
Not sure I follow - I needed to do this for testing to be able to mock `require.resolve`: https://stackoverflow.com/questions/47907454/mocking-and-testing-require-resolve-calls-in-jest

But maybe there is abetter way?

Reply:
You can do it by sth like:

```js
jest.spyOn(require, 'resolve').mockImplementation(() => 'test')
console.log(require.resolve());
require.resolve.mockRestore()